ratpack-bearer-auth/src/main/java/st/ratpack/auth/handler/TokenScopeFilterHandler.java

```diff
@@ -29,7 +30,12 @@ public void handle(Context ctx) throws Exception {
 				sendError(ctx, 403);
```

Comment:
this should be handled by ReqyuireAuthHandler

Reply:
Done